ocaml/tools/ocamlcmt.ml

```diff
@@ -85,14 +85,21 @@ let print_info cmt =
   let compare_imports (name1, _crco1) (name2, _crco2) =
     Compilation_unit.Name.compare name1 name2
   in
+  let imports =
+    cmt.cmt_imports
+    |> Array.to_list
+    |> List.map (fun import ->
```

Comment:
(Same thing here, sorting the
array is likely to be slightly
faster for long lists.)

Reply:
Fixed